0.1 JS Fundamentals -  Introduction/0.1  JS Fundamentals - Introduction.html

```diff
@@ -0,0 +1,41 @@
+<html>
+
+<body>
+    <script>
+        // const arr = [10, 20]
+        // arr.concat([30]);
+
+        // 1.  Declare a variable let age = 25;. Write a series of if else statements that will:
+        //     Print child to the console if age is less than equal to 12.
+        //     Print teen to the console if age is between 13 and 18 (inclusive).
+        //     Print adult to the console if age is above 18
+
+        let age = 99;
+        if (age <= 12) {
+            console.log("Child")
+        }
+        else if (age > 12 && age <= 18) {
+            console.log("Teen")
+        }
+        else {
+            console.log("Adult")
+        }
+
+        // 2.  Do the same using switch case
+
+        switch (age) {
```

Comment:
will this switch case work? Please try to run this code

Reply:
We need to pass true in like this switch(true)